Tools/SandboxTest.ps1

```diff
@@ -219,26 +153,23 @@ function Get-ARPTable {
       Where-Object { $_.DisplayName -and (-not $_.SystemComponent -or $_.SystemComponent -ne 1 ) } |
       Select-Object DisplayName, DisplayVersion, Publisher, @{N='ProductCode'; E={$_.PSChildName}}, @{N='Scope'; E={if($_.PSDrive.Name -eq 'HKCU') {'User'} else {'Machine'}}}
 }
-'@
 
-$bootstrapPs1Content += @"
 Write-Host @'
 --> Installing WinGet
 '@
-`$ProgressPreference = 'SilentlyContinue'
-Add-AppxPackage -Path '$($desktopAppInstaller.pathInSandbox)' -DependencyPath '$($vcLibsUwp.pathInSandbox)','$($uiLibsUwp.pathInSandbox)'
 
+Install-PackageProvider -Name NuGet -Force | Out-Null
+Install-Module -Name Microsoft.WinGet.Client -Force -Repository PSGallery | Out-Null
+# Need to use Start-Job to prevent progress bars from staying on screen. Could do this by setting the progress preference too, but this seemed cleaner
```

Comment:
Are progress bars still displayed though? Wording makes it sound that they still are, but Start-Job prevents it from staying indefinitely. I recall Windows PowerShell downloads get incredibly slow with Progress bars, so you may consider going back to the $ProgressPreference = 'SilentlyContinue' approach?

Related:
- https://github.com/PowerShell/PowerShell/issues/2138
- https://github.com/PowerShell/PowerShell/issues/13414

Reply:
It shows progress bars for installing the PowershellCmdlets, but not for the rest of it